pom.xml

```diff
@@ -200,13 +117,31 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>display-info</id>
+            <configuration>
+              <rules>
+                <enforceBytecodeVersion>
+                  <excludes combine.children="append">
+                    <!-- Loaded conditionally. -->
+                    <exclude>org.eclipse.jgit:org.eclipse.jgit.java7</exclude>
+                  </excludes>
+                </enforceBytecodeVersion>
+              </rules>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
JGit includes a fallback mechanism in case Java 7 is not available (current level is 6).